hadolint/hadolint
# Zypper linting #470

This adds linting rules for `zypper`, so that package installations on OpenSUSE based images can be linted.



### What I did

- Add rules to lint for `zypper` commands. Analogue to `yum` rules
- Update README.md to include the new rules
- Update unit tests to test new rules

### How I did it

Same as with the `yum` rules. Only adapted for `zypper`.

### How to verify it
Good Dockerfile. This should not trigger any of the new rules.
```Dockerfile
FROM opensuse/leap:15.2
RUN zypper install -y tomcat\>=9.2 && zypper clean
```

Bad Dockerfile. This should trigger all four new rules separately
```Dockerfile
FROM opensuse/leap:15.2
RUN zypper install tomcat\>=9.2 && zypper clean
RUN zypper install -y tomcat\>=9.2
RUN zypper install -y tomcat && zypper clean
RUN zypper update -y && zypper clean
